buildkite/bazelci.py

```diff
@@ -567,6 +568,9 @@ def execute_commands(
                 # This will only work if the bazel binary in $PATH is actually a bazelisk binary
                 # (https://github.com/philwo/bazelisk).
                 os.environ["USE_BAZEL_VERSION"] = bazel_version
+                # If the CI worker runs Bazelisk, we need to forward all required env variables to the test.
+                # Otherwise any integration test that invokes Bazel (=Bazelisk in this case) will fail.
+                test_env_vars = ["HOME", "USE_BAZEL_VERSION"]
```

Comment:
In the downstream project pipeline, `--use_but` is passed, so this won't work right?

Reply:
Looks like in downstream project, we are always using `/usr/bin/bazel` inside test instead of the Bazel built from HEAD.

Reply:
Good catch!